onnx/defs/tensor/defs.cc

```diff
@@ -1395,40 +1395,94 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input 'X' and output 'Y' to all tensor types.")
         .SetDoc(Upsample_ver10_doc)
         .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-          resizeShapeInference(ctx);
+          resizeShapeInference(ctx, false);
         }));
 
-static const char* Resize_ver10_doc = R"DOC(
+static const char* Resize_ver11_doc = R"DOC(
 Resize the input tensor.
 Each dimension value of the output tensor is:
-  output_dimension = floor(input_dimension * scale).
+  output_dimension = floor(input_dimension * scale) if input \"sizes\" is not specified.
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Resize,
-    10,
+    11,
     OpSchema()
         .Attr(
             "mode",
-            "Two interpolation modes: nearest (default), and linear (including bilinear, trilinear, etc)",
+            "Three interpolation modes: nearest (default), linear and cubic. "
+            "The \"linear\" mode includes linear interpolation for 1D tensor and N-D linear interpolation for N-D tensor (For example, bilinear interpolation for 2D tensor). "
+            "The \"cubic\" mode includes cubic interpolation for 1D tensor and N-D cubic interpolation for N-D tensor (For example, bicubic interpolation for 2D tensor)."
+            ,
             AttributeProto::STRING,
             std::string("nearest"))
-        .Input(0, "X", "N-D tensor", "T")
+        .Attr("cubic_coeff_a",
+            "The coefficient 'a' used in cubic interpolation. Two common choice are -0.5 (in TensorFlow) and -0.75"
+            " (in PyTorch). Check out https://ieeexplore.ieee.org/document/1163711 for the details.",
+            AttributeProto::FLOAT,
+            static_cast<float>(-0.75))
+        .Attr("exclude_outside",
+            "If set to 1, the weight of sampling locations outside the tensor will be set to 0"
```

Comment:
What is `weight`? Without a equation like 
![image](https://user-images.githubusercontent.com/3524474/62345182-1320d380-b4a6-11e9-84f4-6ea5d0db0768.png)
, it's not easy to explain those attributes in a way that the reader can implement this operator.

Reply:
I have added some description in 1c87d37. Is it enough? Thanks!

Reply:
Looks good to me. Thank you.